File: codeforces/java/src/BearAndBigBrother.java
//  https://codeforces.com/problemset/problem/791/A

import java.util.Scanner;

public class BearAndBigBrother {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int limakWeight = scanner.nextInt();
        int bobWeight = scanner.nextInt();
        
        boolean band = true;
        int years = 0;
        
        while (band) {
            years++;
            limakWeight *= 3;
            bobWeight *= 2;
            if (limakWeight > bobWeight) {
                band = false;
            }
        }
        System.out.println(years);
    }
}

File: codeforces/python/bear_and_big_brother.py
# https://codeforces.com/problemset/problem/791/A

limak_weigth, bob_weigth = map(int, input().split())

band = True
years = 0

while band:
    years += 1
    limak_weigth *= 3
    bob_weigth *= 2
    if limak_weigth > bob_weigth:
        band = False
print(years)    